quickjs-libc.c

```diff
@@ -181,6 +181,16 @@ static BOOL my_isdigit(int c)
     return (c >= '0' && c <= '9');
 }
 
+static JSThreadState *js_get_thread_state(JSRuntime *rt)
+{
+    return (JSThreadState *)js_std_cmd(/*GetOpaque*/0, rt);
+}
+
+static void js_set_thread_state(JSRuntime *rt, JSThreadState *ts)
+{
+    js_std_cmd(/*SetOpaque*/1, rt, ts);
+}
```

Comment:
The other approach is a mutex-protected list or hash table but this seemed all around easier.